go.mod

```
@@ -1,10 +1,10 @@
-module github.com/ajeetdsouza/testcontainers-aerospike-go
+module github.com/bitcoin-sv/testcontainers-aerospike-go
```

bug: updated module path
```suggestion
module github.com/ajeetdsouza/testcontainers-aerospike-go
```